### PR TITLE
Update content/zh/docs/concepts/configuration/secret.md

### PR DESCRIPTION
**Pull request by AllenZMC**
Line 70 , Typo fix: 请参阅 [解码 secret](#decoding-a-secret) 了解如何查看它们的内容。-> 请参阅 [解码 secret](# 解码 Secret) 了解如何查看它们的内容

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/concepts/configuration/secret.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ username.txt:    5 bytes
 
 请注意，默认情况下，`get` 和 `describe` 命令都不会显示文件的内容。这是为了防止将 secret 中的内容被意外暴露给从终端日志记录中刻意寻找它们的人。
 
-请参阅 [解码 secret](#decoding-a-secret) 了解如何查看它们的内容。
+请参阅 [解码 secret](#解码 Secret) 了解如何查看它们的内容。
```
Comment (idealhack) [inline content/zh/docs/concepts/configuration/secret.md]:
```suggestion
请参阅 [解码 secret](#解码-secret) 了解如何查看它们的内容。
```

Comment (xichengliudui) [inline content/zh/docs/concepts/configuration/secret.md]: this is a link, you should not change it

File: content/zh/docs/concepts/configuration/secret.md [hunk 2]
```diff
@@ -583,4 +583,4 @@ Pod 中有多个容器。但是，pod 中的每个容器必须请求其挂载卷
 - 应用程序在从卷中读取 secret 后仍然需要保护 secret 的值，例如不会意外记录或发送给不信任方。
 - 可以创建和使用 secret 的 pod 的用户也可以看到该 secret 的值。即使 API server 策略不允许用户读取 secret 对象，用户也可以运行暴露 secret 的 pod。
 - 如果运行了多个副本，那么这些 secret 将在它们之间共享。默认情况下，etcd 不能保证与 SSL/TLS 的对等通信，尽管可以进行配置。
-- 目前，任何节点的 root 用户都可以通过模拟 kubelet 来读取 API server 中的任何 secret。只有向实际需要它们的节点发送 secret 才能限制单个节点的根漏洞的影响，该功能还在计划中。
\ No newline at end of file
+- 目前，任何节点的 root 用户都可以通过模拟 kubelet 来读取 API server 中的任何 secret。只有向实际需要它们的节点发送 secret 才能限制单个节点的根漏洞的影响，该功能还在计划中。
```
Comment (xichengliudui) [inline content/zh/docs/concepts/configuration/secret.md]: can you tell me what you fixed here?

File: content/zh/docs/concepts/configuration/secret.md (hunk 1 above)
Comment (idealhack) [reply content/zh/docs/concepts/configuration/secret.md]: see https://github.com/kubernetes/website/pull/14687#discussion_r289692040 and https://github.com/kubernetes/website/pull/14690

File: content/zh/docs/concepts/configuration/secret.md (hunk 2 above)
Comment (idealhack) [reply content/zh/docs/concepts/configuration/secret.md]: I guess it's just line break fix by GitHub editor, I'd approve this change

Comment (xichengliudui) [reply content/zh/docs/concepts/configuration/secret.md]: If so, +1

**Pull request closed without merging**